app/src/main/java/com/jerboa/feat/SwipeToAction.kt

```diff
@@ -0,0 +1,95 @@
+package com.jerboa.feat
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Bookmark
+import androidx.compose.material.icons.outlined.Comment
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import com.jerboa.R
+
+enum class SwipeToActionType() {
+    Upvote,
+    Downvote,
+    Reply,
+    Save,
+    ;
+
+    companion object {
+        fun getActionToRangeList(actions: List<SwipeToActionType>): List<Pair<OpenEndRange<Float>, SwipeToActionType>> {
+            val delta = if (actions.size > 2) 0.14f else 0.18f
```

Comment:
Not sure if here is where the issue is, but it switches from Upvote to Downvote way too quickly. Each action needs a longer swipe range (especially the first).